### DISCUSSION TITLE
Parameter tuning for more precision solution

### DISCUSSION THREAD

**Discussion by roccafrancesco** [osqp/osqp#611] [General]
Hello
Is it possible to use this algorithm alone with more precision? Is there a way to rescale all parameters according the eps?
The question is especially relevant if you intend to use this algorithm within any Sequential quadratic solver.
Many problems require precisions of the order 10e-9
Fra

P.S. A Java version, based on paper:"An Operator Splitting Solver for Quadratic Programs(Bartolomeo Stellato, Goran Banjac, Paul Goulart, Alberto Bemporad, Stephen Boyd,February 13 2020) is present on github at link:
https://github.com/Hipparchus-Math/hipparchus/blob/master/hipparchus-optim/src/main/java/org/hipparchus/optim/nonlinear/vector/constrained/ADMMQPOptimizer.java
The code is first issue but thanks to it the development of a module for constrained nonlinear optimization has begun.

**Comment by goulart-paul** [comment]
You can set `eps_rel` to a tighter tolerance, but in general OSQP is not going to produce very high accuracy solutions (at least not quickly) with any consistency just due to the nature of the underlying solver algorithm.      If you want to consistently get solutions with relatively accuracy of 1e-9, I think you would be better off with a different method.

**Comment by roccafrancesco** [comment]
thank you
